internal/components/components.go

```diff
@@ -229,6 +230,7 @@ func Components() (component.Factories, error) {
 		awsxrayexporter.NewFactory(),
 		azuremonitorexporter.NewFactory(),
 		carbonexporter.NewFactory(),
+		clickhouseexporter.NewFactory(),
```

Comment:
Don't forget to open a PR with the releases repo to add this component there: https://github.com/open-telemetry/opentelemetry-collector-releases

Reply:
OK